src/sql/magic_plot.py

```diff
@@ -78,25 +74,30 @@ def execute(self, line="", cell="", local_ns=None):
         Plot magic
         """
 
-        cmd = SQLPlotCommand(self, line)
+        PLOT_STR = util.pretty_print(SUPPORTED_PLOTS, last_delimiter="or")
+        MISSING_LINE_ERROR = (
+            f"Missing the first argument, must be any of: "
+            f"{PLOT_STR}\nExample: %sqlplot histogram"
+        )
+        try:
+            cmd = SQLPlotCommand(self, line)
+        except UsageError as e:
```

Comment:
a try except doesn't seem right here.

I'm guessing the error is raised by `parse.magic_args`? if so, we should encapsulare the validation logic inside SQLPlotCommand, instead of adding the try except here.